src/vivarium_gates_nutrition_optimization_child/components/wasting.py

```diff
@@ -105,6 +106,14 @@ def __init__(self, treatment_type: str):
 
         self.treated_state = self._get_treated_state()
 
+    @property
+    def columns_created(self) -> List[str]:
+        return [self.propensity_column_name]
```

Comment:
Isn't this just the same as the `columns_created` property in `Risk`? You don't need to include this here if so.